packages/backend/src/controllers/users/users.controller.ts

```diff
@@ -23,19 +30,25 @@ export class UsersController {
 
   @Post()
   @ApiOperation({
-    summary: 'create a new user resource',
+    summary: 'Create a new user resource',
     description: 'User must be linked to a firebase ID',
   })
-  @ApiCreatedResponse({
+  @ApiBadRequestResponse({
     description: 'User successfully created',
     type: UserResponseDto,
   })
+  @ApiBadRequestResponse({ description: 'no user name provided' })
   async create(@Body() createUserDto: CreateUserDto): Promise<UserResponseDto> {
     const userDoc = await this.userStoreService.create(createUserDto);
     const houseDoc = await this.houseStoreService.findOne(userDoc.house);
+    const userName = userDoc?.name && createUserDto.name;
```

Comment:
```suggestion
    const userName = userDoc?.name ?? createUserDto.name;
```
&& operator is incorrect here

Reply:
mb extracting logic